src/torchmetrics/functional/classification/stat_scores.py

```diff
@@ -341,6 +341,30 @@ def _multiclass_stat_scores_format(
     return preds, target
 
 
+def refine_preds_oh(preds: Tensor, preds_oh: Tensor, target: Tensor, top_k: int) -> Tensor:
```

Comment:
```suggestion
def _refine_preds_oh(preds: Tensor, preds_oh: Tensor, target: Tensor, top_k: int) -> Tensor:
```
let's rather make it a private function?